KrakenIoc/KrakenIoc/Core/V1/Injector.cs

```diff
@@ -685,27 +685,28 @@ protected virtual List<MemberInfoCache> FindPropertiesRecursive(Type type)
                 {
                     continue;
                 }
-
-                fields.Add(new MemberInfoCache(info, attrs[0]));
+                
+                PropertyInfo strongInfo = info.DeclaringType.GetProperty(info.Name, propertyFlags);
```

Comment:
Please adhere to the script syntax formatting using `var` instead.

Reply:
I missed this. This has been fixed now.